### ISSUE TITLE
6/1/17 Update

### ISSUE THREAD

**Issue by kylemathias** [open-learning-exchange/take-home#58]
I will start making a weekly issue updating people on what went on in the week and what we went over during our weekly Take Home meeting Thursdays.

1. We are still looking for a solution for imaging a Raspberry Pi Image from an android phone. #57 So far we have tried to use an imaging app, but no one has gotten it to work. [Pi SD Card Imager](https://play.google.com/store/apps/details?id=com.redrobe.raspicardimager) 
2. The Raspberry Pi team is looking into modifying our currently used Raspberry Pi Image so it will start with some pre set scripts. This image is what we will be mounting to sd cards with our solution we find. These scripts would then be accessed and run by our second function through a Bluetooth connection to our Android device.
3.  This is a google doc of some links we found to possible solutions to these issues https://docs.google.com/document/d/1fnxF_mV_JjS52h6lKUJg-MQ0tgeGDZV1XcSc3BpLsIc/edit 

So to sum it up, we are still looking into mounting our Image with an Android Device, along with finding out a way to send commands through Bluetooth to the Raspberry Pi.

Other interns, please feel free to post additional topics i might have missed, and post a comment here that you have read this.

Next weeks meeting: #59

**Comment by timecomplexity** [comment]
Posting a comment to show that I have read this. :)

**Comment by Yurockkk** [comment]
I have read this this 😄

**Comment by dogi** [comment]
@kylemathias thanks
